manifest/process.go

```diff
@@ -37,6 +37,16 @@ func NewProcess(app string, s Service, m Manifest) Process {
 		}
 	}
 
+	for i, p := range s.Ports {
+		// Assign random ports to "internal" balancers
+		if !p.Public {
+			p.Public = true
```

Comment:
We should treat manifest as if it is used by both local and production as the eventual goal should be to make that completely true.

Create a docker-compose.yml with `postgres1` and `postgres2` process types and have them both listen on `- 5432` to demonstrate the problem.